test/js/node/child_process/child_process.test.ts

```diff
@@ -214,6 +215,21 @@ describe("spawn()", () => {
     const { stdout } = spawnSync("echo", ["hello"], { encoding: "utf8" });
     expect(stdout.trim()).toBe("hello");
   });
+
+  it("should handle null for first 4 positions of stdio", async () => {
```

Comment:
If we split this out into a `test.if(process.platform === 'linux', (...` then we'll know its skipped in other cases (better visibility and awareness of what is and isn't running for a platform).

Reply:
Done.